activerecord/lib/active_record/connection_adapters/abstract/database_statements.rb

```diff
@@ -315,22 +315,51 @@ def insert_fixture(fixture, table_name)
             raise Fixture::FixtureError, %(table "#{table_name}" has no column named #{name.inspect}.)
           end
         end
+        value_list = binds_to_values(binds)
+
         key_list = fixture.keys.map { |name| quote_column_name(name) }
-        value_list = binds.map(&:value_for_database).map do |value|
-          begin
-            quote(value)
-          rescue TypeError
-            quote(YAML.dump(value))
+        execute "INSERT INTO #{quote_table_name(table_name)} (#{key_list.join(', ')}) VALUES (#{value_list.join(', ')})", "Fixture Insert"
+      end
+
+      DefaultValue = Object.new
```

Comment:
Sure, I'll add one. 
I have to make a workaround here with `DefaultValue` because `Relation::QueryAttribute` can't provide `DEFAULT` as a value. Do you think this workaround is acceptable in this case?